Sources/FluentKit/Query/QueryBuilder+Paginate.swift

```diff
@@ -8,8 +8,15 @@ extension QueryBuilder {
     public func paginate(
         _ request: PageRequest
     ) -> EventLoopFuture<Page<Model>> {
-        let count = self.count()
-        let items = self.copy().range(request.start..<request.end).all()
+        // Remove all eager load requests when counting the rows
+        // otherwise we try to read IDs from the count reply when
+        // performing the eager load subqueries.
+        let countQueryBuilder = self.copy()
+        countQueryBuilder.eagerLoads = .init()
```

Comment:
Maybe this should happen in `count()` itself? 

For example, if you do: 

```swift
var builder = Planet.query(on: db).with(\.$star)
builder.count()
// later
builder.all()
```

You'd get the same error, right?

Reply:
You're correct. I've added it to the aggregate function instead as it's true for all aggregate methods, not just count.